GOVERNANCE.md

```diff
@@ -0,0 +1,94 @@
+
+This document describes a lightweight set of procedures designed to implement this general spirit as effortlessly as possible for the stakeholder involved.
+
+
+# Roles
+
+We only recognize two roles in the UP project:
+
+- **Committer**: a person who can create new branches and commit on any branch except the "master" branch;
+- **Maintainer**: a Committer who can also commit on the "master" branch. A list of all active Maintainers is always kept up-to-date in the [MAINTAINERS](/MAINTAINERS) file.
+
+In addition, being an open source project we welcome external contributions coming from external forks of the UP project. We will refer to such Pull-Requests as "External Contributions".
+
+The **Board of Maintainers** is the main decision body of the UP project and is composed of all the current Maintainers.
+
+
+# Communication Channels
+
+We establish two official communication channels for the UP project in addition to the GitHub facilities (issues, discussions, PRs).
+
+- The **Maintainers Mailing List** (unified-planning-maintainers@googlegroups.com): all and only the active Maintainers are part of this list. The mailing list is not publicly readable but anyone can send messages to it. It is the main and official communication channel among maintainers and to communicate to the board of maintainers.
+
+- The **Public Mailing List** (unified-planning@googlegroups.com): is a general-purpose communication channel to be used for both technical matters and public announcements. The list is publicly readable and anyone can join it. The outcome of all the votes of the Board of Maintainers (required for the situations indicated in the following) are publicly communicated through this list.
+
+On these channels, as well as in any other virtual space associated with the library, the [Unified Planning Library Code of Conduct](/CODE_OF_CONDUCT.md)  applies.
+
+
+# Technical Rules
+
+The following rules apply for the technical day-to-day governance of the project.
+
+1. Any Maintainer can approve and commit to "master" a Pull-Request from any Committer, External Contribution or **another** Maintainer.
+
+2. It is forbidden to anyone to self-approve and commit a Pull-Request.
+
+3. Any Maintainer can reject a Pull-Request from any Committer or External Contribution or Maintainer by publicly motivating the reason in the PR discussion page. Reasons should be of technical nature or concerning the scope of the library and any effort should be made to not waste anybody's work in a constructive manner.
+
+4. If a Maintainer disagrees with another Maintainer decision of accepting or rejecting a Pull-Request, he/she can uphold the final decision to a simple majority vote of the  Board of Maintainers.
```

Comment:
Does this mean that an accepted Pull-Request can be reverted?

Reply:
Good point, added to discussion points